cpp/cmake_modules/CompilerInfo.cmake

```diff
@@ -68,6 +68,11 @@ elseif("${COMPILER_VERSION_FULL}" MATCHES ".*clang-8")
   set(COMPILER_FAMILY "clang")
   set(COMPILER_VERSION "3.8.0svn")
 
+# clang on Mac OS X, XCode 8.
+elseif("${COMPILER_VERSION_FULL}" MATCHES ".*clang-9")
+  set(COMPILER_FAMILY "clang")
+  set(COMPILER_VERSION "4.0.0svn")
```

Comment:
This is just guessing, I have not found any public source that clarifies this. At the moment, we're not using the version, only the family, so this should not matter that much.

Reply:
OK, we can always revisit if this comes up